Blog: August newsletter (#414)
ddev/ddev.com: src/content/blog/ddev-august-2025-newsletter.md

  alt: Nuno Maduro at Laracon US 2025
  caption: "Nuno Maduro at Laracon US 2025"
categories:
  - Community
---

🚀 Dive into August 2025 with DDEV! 🌟  
This month: v1.24.7 release, Contributor Training starts up again, ddev-hostname security, new Windows installer, VS Code extension, DrupalCamp Colorado, Laracon US highlights, Passbolt environment, Tailscale troubleshooting, Magento 2 setup, DrupalCon Vienna, Ollama AI, and Board nominations.

---

## What’s New

- **DDEV v1.24.7 Released** — Includes Windows Installer and MariaDB 11.8 update → [Release notes↗](https://github.com/ddev/ddev/releases/tag/v1.24.7)
- **New Windows Installer Available** — Install DDEV on WSL2 or traditional Windows with the new installer → [Read more↗](https://ddev.com/blog/watch-new-windows-installer)
- **DDEV From Scratch with Windows WSL2** — Step-by-step video guide to get started with DDEV on Windows WSL2 → [Watch guide↗](https://ddev.com/blog/watch-ddev-local-from-scratch-with-windows-wsl2)
- **Security Priorities: Strengthening `ddev-hostname`** — Recent improvements securing host-file updates and future plans → [Read more↗](https://ddev.com/blog/ddev-hostname-security-improvements)

> - **DDEV VS Code Extension v3.0 Released** — New container commands, auto config reload, improved IDE integration, and remote-host support → [Get it on Marketplace↗](https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager)  
>   Learn about “Expose from Container” → [Read more↗](https://github.com/ddev/vscode-ddev-manager/wiki/Expose-from-Container)

Reviewer: ```suggestion
- **DDEV VS Code Extension v3.0 Released** — New container commands, auto config reload, improved IDE integration, and remote-host support → [Get it on Marketplace↗](https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager)  
  Learn about “Expose commands from Container” → [Read more↗](https://github.com/ddev/vscode-ddev-manager/wiki/Expose-from-Container)
```
I am well aware that the docs articles titles is just "Expose from Container"... but maybe i would update the docs article title as well as the line here. problem is in the first sentence it is "new container commands" while in the second sentence just "expose from container". i would ask what is exposed. currently i cant see the connection between those two entries as someone unfamiliar with the topic?

- **DDEV + Magento 2: Quickstart Setup**  
  Automate Magento 2 installs with DDEV. → [View on GitHub↗](https://github.com/jellesiderius/ddev-magento2-setup)
- **Creating a Simple Donation Form in Drupal with Stripe & Webforms**  
  Setup payments in minutes with DDEV and Webform. → [Read more↗](https://picozzi.com/notebook/2025/jul/creating-simple-donation-form-drupal-stripe-and-webforms)
- **Free Local AI with Ollama & DDEV**  
  How to Set Up Ollama as a Free, Local AI Provider for Your local Drupal Development. → [Read more↗](https://www.linkedin.com/pulse/how-set-up-ollama-free-local-ai-provider-your-drupal-rakesh-james-gedje/)

## DDEV Training Starting Up Again!

We're going to do training again this season, all are invited. Some sessions are more focused on contributors and maintainers, and others at all users. First up this month will be a session on using Claude Code to implement a DDEV feature or fix a bug. This will be using DDEV as an example, but probably anybody who works with code anywhere will profit from the approach.

See the full schedule below.

### Upcoming DDEV Live Contributor and User Training Sessions

> 
> ### Upcoming DDEV Live Contributor and User Training Sessions

Reviewer: ```suggestion
```
That line is redundant

  [Download .ics](ics/ddev-2025-10-09.ics)

- **November 20, 2025 at 10:00 US ET / 16:00 CET — Using DDEV in Windows WSL**  
  [Add to Google Calendar](https://calendar.google.com/calendar/render?action=TEMPLATE&text=Using%20DDEV%20in%20Windows%20WSL&dates=20251120T150000Z/20251120T160000Z&details=Join%20the%20DDEV%20training%20session%20via%20Zoom.%0ALink%3A%20https%3A%2F%2Fus02web.zoom.us%2Fj%2F7315692237%3Fpwd%3DRHR6NUkwb0g5WXIzS2NOcXRucCthZz09%0AMeeting%20ID%3A%20731%20569%202237%0APasscode%3A%2012345&location=Online&trp=true) •
  [Download .ics](ics/ddev-2025-11-20.ics)

Zoom Join Info:  
Link: [Join Zoom Meeting](https://us02web.zoom.us/j/7315692237?pwd=RHR6NUkwb0g5WXIzS2NOcXRucCthZz09)  
Passcode: 12345

## Events & Community

- **DrupalCamp Colorado**  

>   I got to see old friends, meet new people, and attend great sessions that refreshed my Drupal knowledge at DrupalCamp Colorado.

Reviewer: ```suggestion
  I got to see old friends, met new people, and attended great sessions that refreshed my Drupal knowledge at DrupalCamp Colorado.
```
shouldnt that be past tense since the the even already took place?

- **Laracon US 2025: Younger, Bigger, Friendlier**  
  Since Laracon was right after DrupalCamp Colorado and in Denver I went to learn from the vibrant Laravel community. It was a big, highly produced event with lots of friendly people and immense emphasis on community. Folks seemed younger on average than in the Drupal world. I didn't learn a lot about Laravel, but there were great general sessions that inspired me to start using Git worktrees and some new AI techniques.
- **See Us at DrupalCon EU in Vienna**  
  Randy will be at [DrupalCon EU in Vienna](https://events.drupal.org/vienna2025) thanks to sponsorship from [Tag1](https://www.tag1consulting.com/) and [Platform.sh/Upsun](https://upsun.com). Please say hi. Let's talk about anything you might want to talk about. I normally do a bunch of Birds-of-a-Feather sessions.

## Governance & Roadmap

- **Informal Nominations for the DDEV Foundation Board**  

>   At the upcoming [DDEV Advisory Group](https://github.com/orgs/ddev/discussions/categories/ddev-advisory-group) meetings we'll be implementing an actual Board of Directors for the DDEV Foundation. Would you like to nominate someone (or yourself)? Ping us via any of our [contact methods](/contact/).

Reviewer: Would it make sense to add the actual date?

Author: Well, it's about both meetings, and the goal is to get some possible nominations.

Reviewer: ```suggestion
  At the upcoming [DDEV Advisory Group](https://github.com/orgs/ddev/discussions/categories/ddev-advisory-group) meetings in September and November, we'll be implementing an actual Board of Directors for the DDEV Foundation. Would you like to nominate someone (or yourself)? Ping us via any of our [contact methods](/contact/).
```
oh two meetings? i've missed that skimming. so how about adding that detail about the months at least? cuz point is that line was about the meetings, there was no direct date given so i've assumed the date would be listed on the github page, but on there i was unable to find an entry for any upcoming meeting. by adding the months it would be clearer that there are two meetings and when they roughly take place. just a minor change, cuz in the current state it lead me as the nosey reader to some dead end. 

oh two meetings.
